gsfpy/bindings.py

```diff
@@ -23,9 +24,22 @@
 from .gsfScaleFactors import c_gsfScaleFactors
 from .gsfSwathBathyPing import c_gsfSwathBathyPing
 
-_gsf_lib_rel_path = "libgsf/libgsf03-08.so"
-_gsf_lib_abs_path = path.join(path.abspath(path.dirname(__file__)), _gsf_lib_rel_path)
-_gsf_lib = CDLL(_gsf_lib_abs_path)
+_libgsf_abs_path = str(Path(__file__).parent / "libgsf" / "libgsf03-08.so")
+
+# Check if the libgsf shared object library location is specified in the environment.
+# If so, use the specified library in preference to the bundled version. Handle the
+# case where the library cannot be found.
+if "GSFPY_LIBGSF_PATH" in environ:
+    _libgsf_abs_path = environ["GSFPY_LIBGSF_PATH"]
+
+try:
+    _gsf_lib = CDLL(_libgsf_abs_path)
```

Comment:
`_gsf_lib` should be renamed to `_libgsf`

Reply:
Done